javascript/js-modern-syntax/js-modern-syntax_destructuring-objects/index.js

```diff
@@ -49,17 +57,22 @@ const logInfo = (city) => {
   const country = city.country;
   const numPeople = city.population;
 
-  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`
-}
+  return `${name} is in ${country} and has ${numPeople} inhabitants in it.`;
+};
+
+const { name, country, numPeople } = city;
```

Comment:
This exercise was a bit confusing..it took me a while till I understood the intend of it: Line 55- 61 were meant to be an example for the actual task starting in line 70. Therefore, your solution should go to line 70.

The other issue with your solution is, that `numPeople` doesn't exist yet. Before using the variable `numPeople` you need to declare and assign `city.population` to it: 

~~`{ name, country, numPeople }`~~ ➔ `{ name, country, population: numPeople }`